examples/device.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+#
+# A simple example of how to use the API.
+
+require 'rubygems'
+require 'sift'
+require 'multi_json'
+
+TIMEOUT = 5 
+api_key = ENV['API_KEY']
+account_id = ENV['ACCOUNT_ID']
+
+
+if api_key.nil? || account_id.nil?
+  puts "Please specify the environment variables API_KEY and API_ID"
+  puts "usage:"
+  puts " API_KEY=xxxxxx API_ID=xxxxxxxx ruby device_id.rb"
+  puts
+  exit
+end
+
+class MyLogger
+  def warn(e)
+    puts "[WARN]  " + e.to_s
+  end
+
+  def error(e)
+    puts "[ERROR] " + e.to_s
+  end
+
+  def fatal(e)
+    puts "[FATAL] " + e.to_s
+  end
+
+  def info(e)
+    puts "[INFO]  " + e.to_s
+  end
+end
+
+Sift.logger = MyLogger.new
+Sift.api_key = api_key
+Sift.account_id = account_id
+
+def print_device(d)
+  puts 'i got the device obj: ' + d.inspect
+  puts 'device bad? ' + d.bad?.to_s
+  puts 'device labeled? ' + d.labeled?.to_s
+  puts 'device labeled_at ' + d.labeled_at.to_s
+  puts 'device session timestamps ' + d.session_timestamps.join(',')
+end
+
+device = Sift::Device.retrieve("i08a78q46cekukrugj7ltnqskj")
```

Comment:
Should this (and the session id) be specifiable on the command line?

More broadly, do we expect users to run this program or just use it as an example?